homeassistant/components/knx/websocket.py

```diff
@@ -220,19 +222,19 @@ def ws_subscribe_telegram(
     msg: dict,
 ) -> None:
     """Subscribe to incoming and outgoing KNX telegrams."""
-    knx: KNXModule = hass.data[DOMAIN]
 
     @callback
-    def forward_telegram(telegram: TelegramDict) -> None:
+    def forward_telegram(_telegram: Telegram, telegram_dict: TelegramDict) -> None:
         """Forward telegram to websocket subscription."""
         connection.send_event(
             msg["id"],
-            telegram,
+            telegram_dict,
         )
 
-    connection.subscriptions[msg["id"]] = knx.telegrams.async_listen_telegram(
-        action=forward_telegram,
-        name="KNX GroupMonitor subscription",
+    connection.subscriptions[msg["id"]] = async_dispatcher_connect(
```

Comment:
Do we unsubscribe from these btw?

Reply:
I would have assumed this happens automatically. It uses `hass.connection.subscribeMessage` in FE.

Reply:
No i mean, do we unsubscribe from the dispatcher in the backend?

Reply:
This happens here https://github.com/home-assistant/core/blob/dev/homeassistant/components/websocket_api/connection.py#L238

Reply:
Awesome!